IAB Workshop.md

```diff
@@ -26,8 +31,13 @@ in public sources like Sonic or FRR?
 Would it help to identify a set of models that belong to a YANG library that are known to work together?
 What will constitute a minimal set of models that should be known to work together? 
 
+# What this workshop would not cover
+- encryption in operator environments
+- what else?
+
 # Outreach
 In 2002, there were enough operators attending IETF meetings. Those numbers have dwindled over time, with some of the operators attending conferences that are more operator focused. These include RIPE, NANOG, AutoConn etc.
 
-As such the IAB may have to do an outreach effort to get the input they are looking for. A tentative plan is to target these conferences in the October/November 2024 timeframe, followed by a virtual workshop in December to discuss the findings. 
+As such the IAB may have to do an outreach effort to get the input they are looking for. A tentative plan is to target these conferences in the October/November 2024 timeframe, and virtual workshops, followed by a final virtual calls tentatively in December to discuss the findings. 
```

Comment:
Were we planning multiple virtual workshops after the conferences, and multiple virtual calls in December? I thought Wes mentioned one.

Reply:
I think that's up for debate and we should plan that in a larger meeting.  Anything is possible depending on what we want to commit to doing :-)